lib/config/presets/util.ts

```diff
@@ -51,7 +51,9 @@ export async function fetchPreset({
   } else {
     jsonContent = await fetch(
       repo,
-      buildFilePath(`${fileName}.json`),
+      buildFilePath(
+        fileName.endsWith('.json5') ? fileName : `${fileName}.json`
```

Comment:
i like to also support explicit json extension in this PR, should be easy to implement with a regex.